projects.yaml

```diff
@@ -554,6 +554,13 @@ projects:
   category: datasets
   description: Database of yet-to-be-sythesized materials predicted using state-of-the-art machine learning algorithms
   homepage: https://matterverse.ai/
+- name: Open Databases Integration for Materials Design (OPTIMADE)
+  resource: true
+  category: datasets
+  description: Specification of a common REST API for access to 25+ materials databases
```

Comment:
Will remove `description`. The specified repo `Materials-Consortia/OPTIMADE` has a description (the same). CI/CD will fetch that. Only provide description if repo does not have (a reasonably informative) one, or if the project is a resource.